pkg/manifests/manifests.go

```diff
@@ -122,15 +120,15 @@ func (f *Factory) RouterServiceCloud(cr *ingressv1alpha1.ClusterIngress) (*corev
 
 	s.Name = name
 
-	s.Labels = map[string]string{
-		"app":    "router",
-		"router": name,
+	if s.Labels == nil {
+		s.Labels = map[string]string{}
 	}
+	s.Labels["router"] = name
```

Comment:
We should update `assets/router/service-cloud.yaml` to reflect that only the one field, and not the entirety of `.metadata.labels`, is replaced at runtime.  Ditto for `spec.selector`.